src/app/components/Main/Main.css

```diff
@@ -1,3 +1,9 @@
 .main-home-page {
```

Comment:
1) Instead of adding bigger margin-bottom to every .populat-playlist-container it's better to make flexbox and make a gap between elemnts in main.
In another way: we see scroll bar really far away from actually list of playlist covers. See on the picture
<img width="682" alt="Screenshot 2023-11-12 at 20 13 04" src="https://github.com/LorenaCapraru/song-sieve/assets/91835307/54bb5cb2-76df-4c29-8e43-7f4133200437">
```suggestion
.main-home-page {
  padding: 30px 10px;
  display: flex;
  flex-direction: column;
  gap: 30px;
}

@media (min-width: 768px) {
  .main-home-page {
    padding: 0 25px;
  }
}
```

2) Instead of making padding in %, I made already padding for header in px: for desktop: 25px - right and left. And for mobile it will be 10px.
In this way we make sure that elements in header and main are on the same level:)

Reply:
genius, thanks Vitalina!